api/ipsec/update

```diff
@@ -49,11 +49,14 @@ if ($cmd eq 'update') {
     }
 
     $db->set_prop($input->{'name'}, 'left', '%'.$input->{'left'});
-} elsif ($cmd eq 'enable' || $cmd eq 'disable') {
-    my $status =( $cmd eq 'enable') ? 'enabled' : 'disabled';
-    $db->set_prop($input->{'name'}, 'status', $status);
+    system(("/sbin/e-smith/signal-event", "-j", "nethserver-ipsec-tunnels-save"));
+} elsif ($cmd eq 'enable') {
+    $db->set_prop($input->{'name'}, 'status', 'enabled');
+    system(("/sbin/e-smith/signal-event", "-j", "nethserver-ipsec-tunnels-save"));
+} elsif ($cmd eq 'disable') {
+    $db->set_prop($input->{'name'}, 'status', 'disabled');
+    my $remoteIP = $db->get_prop($input->{'name'}, 'right');
+    system(("/sbin/e-smith/signal-event", "-j", "nethserver-ipsec-tunnels-modify", "disable", $input->{'name'}, $remoteIP));
```

Comment:
We always need a full ipsec restart, see https://github.com/NethServer/nethserver-ipsec-tunnels/pull/15#pullrequestreview-566951198

You could pass the remote IP to the `nethserver-ipsec-tunnels-save` event and add an actions inside it.
The actions could delete the connections only if the remoteIP param is specified.

Reply:
I think I'm wrong.
This is not going to work because inside the event, the service will be restarted after the connection is deleted.
And if a network packet is sent or received before ipsec restart, the connection will be restored inside the conntrack.

As alternative solution, we could add a script inside the `ExcecPre` of `ipsec` systemd unit.